### DISCUSSION TITLE
Default values for fields in models

### DISCUSSION THREAD

**Discussion by Disregard-Therest** [serverpod/serverpod#1984] [Q&A] [answered]
Hi, please, help, I can't find how to declare default values for fields.

Simplest example - I need a bool field which should be false by default always, but I don't want to set it manually from every point where I create this model

**Comment by SandPod** [comment, marked as answer]
Hi,

Unfortunately Serverpod does not support default values yet, but we have an issue that tracks adding it here: https://github.com/serverpod/serverpod/issues/1140
